test/ERC20Channel.test.js

```diff
@@ -0,0 +1,130 @@
+    const hash = await tokenChannels.generateBalanceHash(accounts[1], nonce, 20);
+    const senderSig = signMsg(hash, privateKeys[accounts[2]]);
+    assert.equal(accounts[2],
+      await tokenChannels.getSignerOfBalanceHash(accounts[1], nonce, 20, senderSig)
+    );
+    const senderHash = await tokenChannels.generateKeccak256(senderSig);
+    const closingSig = signMsg(senderHash, privateKeys[accounts[1]]);
+    await tokenChannels.cooperativeClose(accounts[1], nonce, 20, senderSig, closingSig, { from: accounts[2] });
+    (await token.balanceOf(accounts[2])).should.be.bignumber
+      .equal(80);
+    (await token.balanceOf(accounts[1])).should.be.bignumber
+      .equal(20);
+  });
+
+  it('create channel and close it from sender with uncooperativeClose', async function () {
+    await token.approve(tokenChannels.address, 30, { from: accounts[2] });
+    const nonce = 33;
+    await tokenChannels.openChannel(accounts[1], 30, nonce, { from: accounts[2] });
+    const hash = await tokenChannels.generateBalanceHash(accounts[1], nonce, 10);
+    const senderSig = signMsg(hash, privateKeys[accounts[2]]);
+    assert.equal(accounts[2],
+      await tokenChannels.getSignerOfBalanceHash(accounts[1], nonce, 10, senderSig)
+    );
+    await tokenChannels.uncooperativeClose(accounts[1], nonce, 10, { from: accounts[2] });
+    try {
+      await tokenChannels.closeChannel(accounts[1], nonce, { from: accounts[2] });
+      assert.fail('should have thrown before');
+    } catch (error) {
+      assertRevert(error);
+    }
+    await increaseTimeTo(latestTime() + 61);
+    await tokenChannels.closeChannel(accounts[1], nonce, { from: accounts[2] });
+    (await token.balanceOf(accounts[2])).should.be.bignumber
+      .equal(90);
+    (await token.balanceOf(accounts[1])).should.be.bignumber
+      .equal(10);
+  });
+
+  it('create channel and close it from receiver after sender chanllenge with different balance', async function () {
+    await token.approve(tokenChannels.address, 30, { from: accounts[2] });
+    const nonce = 33;
+    await tokenChannels.openChannel(accounts[1], 30, nonce, { from: accounts[2] });
+    const hash = await tokenChannels.generateBalanceHash(accounts[1], nonce, 20);
+    const senderSig = signMsg(hash, privateKeys[accounts[2]]);
+    assert.equal(accounts[2],
+      await tokenChannels.getSignerOfBalanceHash(accounts[1], nonce, 20, senderSig)
+    );
+    const senderHash = await tokenChannels.generateKeccak256(senderSig);
+    const closingSig = signMsg(senderHash, privateKeys[accounts[1]]);
+    await tokenChannels.uncooperativeClose(accounts[1], nonce, 10, { from: accounts[2] });
+    await increaseTimeTo(latestTime() + 10);
+    await tokenChannels.cooperativeClose(accounts[1], nonce, 20, senderSig, closingSig, { from: accounts[1] });
+    (await token.balanceOf(accounts[2])).should.be.bignumber
+      .equal(80);
+    (await token.balanceOf(accounts[1])).should.be.bignumber
+      .equal(20);
```

Comment:
Could you add an assertion, in all tests that end with a successful close, that the call to `getChannelInfo` returns empty?